chromium_src/chrome/browser/ui/bookmarks/bookmark_utils.cc

```diff
@@ -50,11 +50,17 @@ void BraveToggleBookmarkBarState(content::BrowserContext* browser_context) {
   }                                                                        \
   void ToggleBookmarkBarWhenVisible_ChromiumImpl
 
+#if defined(TOOLKIT_VIEWS)
```

Comment:
Do we need this guard? W/o this, there is a build failure?
I think it's not necessary because this file `bookmark_utils.cc` is in `//chrome/browser/ui/bookmarks:impl` target.
and that target is only added in deps of non-android `ui` target.

Reply:
Didn't know that but I think we'd better to have this because the upstream's GetBookmarkFolderIcon() is guarded with TOOLKIT_VIEWS in the source code.

Reply:
ok, their guard was added in 10y ago. I think it should be cleaned up.